web/app/components/custom-editable-field.hbs

```diff
@@ -1,60 +1,11 @@
-{{#if this.typeIsString}}
+<div class="editable-field-container" ...attributes>
   <EditableField
-    data-test-custom-string-field
-    @value={{get @document @field}}
-    @onChange={{@onChange}}
+    data-test-custom-field
+    data-test-custom-field-type={{if this.typeIsString "string" "people"}}
+    @value={{if this.typeIsString this.stringValue this.hermesUsers}}
+    @onChange={{if this.typeIsPeople this.onPeopleSelectChange}}
+    @onSave={{if this.typeIsPeople this.onPeopleSave @onSave}}
```

Comment:
Not necessary for this PR, but it might be good to get ahead of other potential types besides "string" and "people" sooner than later, and handle this in a less binary way.